Lib/test/test_ctypes/test_dllist.py

```diff
@@ -0,0 +1,72 @@
+import os
+import sys
+import test.support
+import unittest
+from ctypes import CDLL
+from ctypes.util import dllist
+
+
+WINDOWS = os.name == 'nt'
+APPLE = sys.platform in {"darwin", "ios", "tvos", "watchos"}
```

Comment:
It's worth noting that Python also supports Android, see [PEP 738](https://peps.python.org/pep-0738). I don't know all that much about Android -- do we need an extra case for the system library, or is it just `libc.so`?